File-Formats/installation-and-upgrade/System-Requirements.md

```diff
@@ -29,19 +29,13 @@ Syncfusion File Formats platform can be installed on any of the following operat
 * openSUSE: 15+
 * SUSE Enterprise Linux (SLES): 12 SP2+
 * Mac OS 10.13+ (x64)
-
-### Hardware Environment
-
-* Processor: x86 or x64
-* RAM : 1 GB (minimum), 2 GB (recommended)
-* Hard disc: up to 8.5 GB of available space may be required. However, 750 MB free space is required in boot drive even if you are installing in other drive.
+* Android
+* iOS
 
 ### Development Environment
 
-In order to develop applications with Syncfusion File Formats platform, you need to have installed anyone of the Visual Studio version.
+In order to develop applications with Syncfusion File Format libraries, you need to have installed anyone of the Visual Studio version.
```

Comment:
In order to develop applications with the Syncfusion File Format libraries, you need to install any of the Visual Studio versions.

Reply:
Addressed